src/gui/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ set(srcs
     main.cpp
     mainwindow.cpp
     mainwindow.ui
+    PropertyModel.cpp
```

Comment:
please align with naming scheme i.e. lowercase names.

Reply:
What is the reason of this non-standard naming scheme, i.e. file names not matching names of class defined in them?

Reply:
Hah, good question. I suppose we have a mix of an official C++ style (and thinking), and Java-oriented thinking (to which Qt sticks quite rigidly) ;-)